src/hotspot/cpu/x86/vm_version_x86.cpp

```diff
@@ -1174,6 +1176,18 @@ void VM_Version::get_processor_features() {
     FLAG_SET_DEFAULT(UseSHA256Intrinsics, false);
   }
 
+#ifdef _LP64
+  if (supports_avx512ifma() & supports_avx512vlbw()) {
+    if (FLAG_IS_DEFAULT(UsePolyIntrinsics)) {
```

Comment:
MaxVectorSize > 32 can be added along with feature checks.

Reply:
done (`MaxVectorSize >= 64`)